twomes/measurement.go

```diff
@@ -2,9 +2,10 @@ package twomes
 
 // A Measurement is a measured value for a specific property.
 type Measurement struct {
-	ID       uint     `json:"id"`
-	UploadID uint     `json:"upload_id"`
-	Property Property `json:"property"`
-	Time     Time     `json:"time"`
-	Value    string   `json:"value"`
+	ID         uint     `json:"id"`
+	UploadID   uint     `json:"upload_id"`
+	PropertyID int      `json:"-"`
```

Comment:
Why add this here? It's in Property already and you hid it in a JSON response?

Reply:
Without this, I get below error.

```
twomes-backoffice-api-local-web  | time="2023-06-02T08:22:55Z" level=info msg="\"GET http://localhost:8080/device/TWOMES-9799B8/measurements?start=2023-05-09T12%3A33%3A36.324Z HTTP/1.1\" from 172.24.0.1:65494 - 500 36B in 11.471525ms"
twomes-backoffice-api-local-web  | 
twomes-backoffice-api-local-web  | 2023/06/02 08:22:55 /go/src/twomes-api-server/repositories/device.go:108
twomes-backoffice-api-local-web  | [error] invalid field found for struct github.com/energietransitie/twomes-backoffice-api/twomes.Measurement's field Property: define a valid foreign key for relations or implement the Valuer/Scanner interface
```

Reply:
Thanks!